static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+
+import {AreaChart, AreaChartProps} from 'sentry/components/charts/areaChart';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {formatBytesBase2} from 'sentry/utils';
+import theme from 'sentry/utils/theme';
+
+function MemoryChart({memorySpans}) {
```

Comment:
eslint fix after accepting this one, but we should type our props:

```suggestion
import type {RawSpanType} from 'sentry/components/events/interfaces/spans/types';

type Props = {
  memorySpans: RawSpanType[]
};

function MemoryChart({memorySpans}: Props) {
```